src/MultiCoinNetwork.php

```diff
@@ -22,7 +22,7 @@ class MultiCoinNetwork extends Network {
      * @param string $symbol
      * @param string $network  [main, test, regtest]
      */
-    function __construct($symbol, $network='main') {
+    function __construct(string $symbol, string $network='main') {
```

Comment:
please drop the type hints.   not relevant to this fix, and requires php7, but composer.json requires php 5.6+